edgelet/iotedge/src/error.rs

```diff
@@ -44,6 +44,12 @@ pub enum ErrorKind {
 
     #[fail(display = "Could not write to file")]
     WriteToFile,
+
+    #[fail(display = "Unable to bundle iotedge check")]
```

Comment:
I think all of these (including `WriteToFile`) could be merged into `#[fail(display = "Could not generate support bundle")] SupportBundle`. Are the inner errors good enough to figure out what substep failed?

Reply:
They are readable. Plus the errors for the individual components (logs, check, inspect) don't fail the command, just note the error in the console and output file. So if inspect fails, you'll still get the logs and check output.